polaris-react/src/components/ButtonGroup/ButtonGroup.scss

```
@@ -90,3 +90,8 @@
     margin-left: var(--p-space-5);
   }
 }
+
+.noWrap {
+  display: flex;
+  flex-wrap: nowrap;
+}
```

I think the bug highlighted by the visual regression snapshots can be fixed by moving this class in with the rest of the subclasses 👀 

```suggestion

  .noWrap {
    display: flex;
    flex-wrap: nowrap;
  }
}
```

---

So, moving this up into the above brackets would move it inside the `.loose` class, no?

---

If I look at the `.scss` file, none of these sub classes actually seem to be inside `.ButtonGroup`

---

Ah yeah sorry! Made the suggestion on scanning the file here, but no the subclasses aren't nested so there's something else going on 👀 It's odd because the .segmented class also makes the button group not wrap so I'd expect what you have to be enough, but I haven't had time to check out the branch and poke around.

---

I played around with this in the Playground and it seems to be working as expected now that I've moved the class into the proper div on the `tsx` page.  I also updated the `ButtonGroup NoWrap` story to show the difference between default and nowrap.  Let me know if that can be improved.  Thanks @chloerice!